### DISCUSSION TITLE
Command question

### DISCUSSION THREAD

**Discussion by WinglessWhistle** [jagrosh/MusicBot#887] [Questions]
Is it possible to have something like !leave so that I can set the bot to not leave instantly and instead be made to leave.

**Comment by fanesz** [comment]
yes, got to config.txt, change
stayinchannel = false
to
stayinchannel = true

also you can use [prefix]stop to disconnecting bot
and change the alias in config.txt
  stop = [ leave ]
so you can use !leave to disconnect the bot

note : if u don't know about the config.txt file, simply make empty config.txt and move it to your system32 folder, and then run the jar file, after that the config.txt will be filled with anything that you need!

**Reply by MichailiK** [reply]
> note : if u don't know about the config.txt file, simply make empty config.txt and move it to your system32 folder, and then run the jar file, after that the config.txt will be filled with anything that you need!

Don't do this. This is a bug that should have been patched a long time ago. I patched it again recently & the fix should be coming in 0.3.6 soon (#824). A fix is to set Java the default app for .jar files.

**Reply by fanesz** [reply]
what kind a bug btw? i do that and work perfectly

**Reply by YJSoft** [reply]
@Vanezkun It was loading files from `system32` folder(which is not intended at all)
So you should put config.txt file at java file's path. Maybe it's working for now, but when next version is released, that will stop working.
